siatest/renter.go

```diff
@@ -116,9 +117,9 @@ func (tn *TestNode) Upload(lf *LocalFile, dataPieces, parityPieces uint64) (*Rem
 }
 
 // UploadNewFile initiates the upload of a filesize bytes large file.
-func (tn *TestNode) UploadNewFile(filesize int, dataPieces uint64, parityPieces uint64) (rf *RemoteFile, err error) {
+func (tn *TestNode) UploadNewFile(filesize int, dataPieces uint64, parityPieces uint64) (lf *LocalFile, rf *RemoteFile, err error) {
```

Comment:
Same thing as the other PR, we really prefer to avoid named return values where we can, we've found it causes mistakes especially in longer functions. Sometimes it's just the easiest way to do things, but for longer functions it causes bugs. For example, a blank `return` halfway through the function can change meaning if you add another return value or change the order in which you do things in the function, and that change may not even appear in the diff for reviewers because the `return` value itself is buried somewhere between the lines that changed.